src/components/messageSpace/messageSpace.tsx

```diff
@@ -13,14 +15,87 @@ export interface MessageSpaceProps extends MessageContainerProps {
   /** A component map contains message formats as keys and their corresponding React components as values. */
   supportedElements: ComponentMap
   messages?: ThreadableMessage[]
+  /** Text label for scroll down button. Default value is 'scroll down'. */
+  scrollDownLabel?: string
 }
 
 /**
  The `MessageSpace` component uses `MessageCanvas` and `ElementRenderer` to render a list of messages. It serves as a container for individual message items, each encapsulated within a `MessageCanvas` for consistent styling and layout. \n\n Note: For more information about the `getActionsComponent` and `getProfileComponent` fields, refer to the [MessageCanvas' docs](http://localhost:6006/?path=/docs/rustic-ui-message-canvas-message-canvas--docs).
  */
 export default function MessageSpace(props: MessageSpaceProps) {
+  const scrollEndRef = useRef<HTMLDivElement>(null)
+  const messageContainerRef = useRef<HTMLDivElement>(null)
```

Comment:
I think messageContainer may be confused with the messageCanvas. It might be better to use just containerRef or componentRef or something similar. 

```suggestion
  const containerRef = useRef<HTMLDivElement>(null)
```